src/containers/Tenant/Diagnostics/Diagnostics.scss

```diff
@@ -1,3 +1,9 @@
+.yc-loader {
+    flex-grow: 1;
+    justify-content: center;
+    align-items: center;
+}
```

Comment:
now here is dangerous stuff
if you need to tweak styles for an element, give it its own classname
or at least nest it in the component selector (own classname still wins)
this one alters the styles for all loaders, possibly on other pages as well, since all styles are bundled